src/game_api/script/usertypes/state_lua.cpp

```diff
@@ -141,7 +141,11 @@ void register_usertypes(sol::state& lua)
         "items",
         &StateMemory::items,
         "camera_layer",
-        &StateMemory::camera_layer);
+        &StateMemory::camera_layer,
+        "get_correct_ushabti_af",
+        &StateMemory::get_correct_ushabti_af,
+        "set_correct_ushabti_af",
+        &StateMemory::set_correct_ushabti_af);
```

Comment:
How about just writing it out in full so no one has to guess what `af` means?

Reply:
I'm not a fan of long names for functions :/

Reply:
If you don't want long function names then remove the `_af` suffix and properly document it